docs/domain_model_v2/definitions/framework_definition.json

```diff
@@ -72,6 +72,18 @@
             "defaultValue": "Live",
             "renderingHints": "{'inputType': 'select', 'order': 5}"
         },
+        {
+            "propertyName": "translations",
```

Comment:
Translation map keys should be valid language codes. Any changes to the `translations` property should check for this validation.

Reply:
Changes done to Validate Language Code